src/ray/rpc/worker/direct_actor_client.h

```diff
@@ -115,14 +118,14 @@ class DirectActorClient : public std::enable_shared_from_this<DirectActorClient>
   ClientCallManager &client_call_manager_;
 
   /// Queue of requests to send.
-  std::deque<std::pair<std::unique_ptr<PushTaskRequest>, ClientCallback<PushTaskReply>>>
-      send_queue_;
+  std::deque<std::pair<std::unique_ptr<PushTaskRequest>,
+                       ClientCallback<PushTaskReply>>> send_queue_ GUARDED_BY(mutex_);
 
   /// The number of bytes currently in flight.
-  int64_t rpc_bytes_in_flight_ = 0;
+  int64_t rpc_bytes_in_flight_ GUARDED_BY(mutex_);
```

Comment:
Hm maybe = 0 after the annotation would work?

Reply:
Oh yeah, that works.